fe/fe-core/src/main/java/org/apache/doris/datasource/hive/HiveMetadataOps.java

```diff
@@ -186,6 +186,12 @@ public void commit(String dbName,
         Table table = client.getTable(dbName, tableName);
         HMSCommitter hmsCommitter = new HMSCommitter(this, fs, table);
         hmsCommitter.commit(hivePUs);
+        ExternalDatabase<?> db = catalog.getDbNullable(dbName);
+        if (db == null) {
+            LOG.warn("Failed to get database: '" + dbName + "' in catalog: " + catalog.getName());
+        } else {
+            db.setUnInitialized(true);
```

Comment:
2 questions:
1. Why refresh the db, not table?
2. Should call `env.getCatalogMgr().refreshExternalTable`, because the refresh info need to sync to other FE via editlog.re